.github/workflows/ci.yml

```diff
@@ -6,16 +6,12 @@ on:
     types: [opened, synchronize]
 
 env:
-  REPO_NAME_SLUG: cowswap
-  PR_NUMBER: ${{ github.event.number }}
   NODE_VERSION: lts/gallium
   REACT_APP_SENTRY_DSN: ${{ secrets.SENTRY_DSN }}
   REACT_APP_PINATA_API_KEY: ${{ secrets.REACT_APP_PINATA_API_KEY }}
   REACT_APP_PINATA_SECRET_API_KEY: ${{ secrets.REACT_APP_PINATA_SECRET_API_KEY }}
-  REACT_APP_SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
-  REACT_APP_BLOCKNATIVE_API_KEY: ${{ secrets.REACT_APP_BLOCKNATIVE_API_KEY }}
   REACT_APP_GOOGLE_ANALYTICS_ID: ${{ secrets.REACT_APP_GOOGLE_ANALYTICS_ID }}
-  REACT_APP_AMPLITUDE_KEY: ${{ secrets.REACT_APP_AMPLITUDE_KEY }}
+  REACT_APP_BLOCKNATIVE_API_KEY: ${{ secrets.REACT_APP_BLOCKNATIVE_API_KEY }}
```

Comment:
but why? we need to convert from the secrets to envs, right?

Reply:
Yes, but only if they are used.
Since the building is all happening inside the `deployment` workflow, they are probably not needed here anymore.
Unless there's a job that still needs it like Cypress. That's why I'm not sure.